src/components/StatusIcon/StatusIcon.tsx

```diff
@@ -16,7 +15,7 @@ const icons = {
     [EFlag.Blue]: CircleInfoFillIcon,
     [EFlag.Yellow]: CircleExclamationFillIcon,
     [EFlag.Orange]: TriangleExclamationFillIcon,
-    [EFlag.Red]: CircleXmarkFillIcon,
+    [EFlag.Red]: CircleExclamationFillIcon,
```

Comment:
Because XMark icon gives wrong associations. We decided to change icons completely, but new set of icons from our designer is not ready yet. So the same icon but with different colors as temporary solution